src/Tribe/Event_Repository.php

```diff
@@ -57,18 +106,18 @@ public function filter_by_cost( $value, $operator = '=', $symbol = null ) {
 			'NOT BETWEEN',
 			'IN',
 			'NOT IN',
-		) ) ) {
+		] ) ) {
 			throw Tribe__Repository__Usage_Error::because_this_comparison_operator_is_not_supported( $operator, 'filter_by_cost' );
 		}
 
-		if ( in_array( $operator, array(
+		if ( in_array( $operator, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: be1684730bd2ed5c3f0106a90c6a6bd5</sub>